### DISCUSSION TITLE
#10. VATZ biweekly meeting at 2022-06-10 13:00:00(KST)

### DISCUSSION THREAD

**Discussion by xellos00** [dsrvlabs/vatz#136] [General]
Retrospect: Sprint 9 
![image](https://user-images.githubusercontent.com/6308023/170710802-e4d0c113-9e2d-4603-8388-60fded9c9733.png)
![image](https://user-images.githubusercontent.com/6308023/170711920-3d8dd491-6455-47e9-a516-c436002cd410.png)
> 8/20
- Finished
   - #80 
   - #108 
   - #113 
   - #114 
   - #117 
   - #123 
   - #129 
   - #131    
 
 - Push to Sprint 10
    - #69 
    - #85 
    - #87 
    - #96 
    - #98 
    - #101 
    - #118 
    - #119 
    - #120 
    - #121 
    - #122 
    - #127 

---
# Raise hand to discuss

- https://github.com/dsrvlabs/vatz/pull/138
- https://github.com/dsrvlabs/vatz/pull/139

**Comment by gnongs** [comment]
## 진홍
- #127

**Reply by xellos00** [reply]
Skip for next Sprint

**Comment by heejin-github** [comment]
# 희진
- Development
  - #118 
    - https://github.com/dsrvlabs/vatz-plugin-solana/pull/4
  - #140 
    - #141

**Comment by xellos00** [comment]
# 동요

- https://github.com/dsrvlabs/vatz/issues/98
   - https://github.com/dsrvlabs/vatz-plugin-near
- https://github.com/dsrvlabs/vatz/issues/96

**Comment by rootwarp** [comment]
# 준교
- #137 
- #133 
- #122

**Comment by kim201212** [comment]
# 형기

I couldn't develop vatz because I had other work.

**Reply by hqueue** [reply]
I confirm that @kim201212 is busy with Ethereum upgrades recently :)

**Comment by meetrick** [comment]
# 황제
- https://github.com/dsrvlabs/vatz-plugin-osmosis
  - check the uptime

**Reply by xellos00** [reply]
Prevent to execute call in plugin we should use go lib and not to use EXEC as much as possible.

**Reply by hqueue** [reply]
If plugin required something, it should be described in the document :)

e..g. Requires xxx software
